api/src/main/java/org/eclipse/microprofile/reactive/messaging/Acknowledgment.java

```diff
@@ -87,10 +57,11 @@
     PRE_PROCESSING,
 
     /**
-     * Acknowledgment performed automatically after the user processing of the message.
+     * Acknowledgment performed automatically after the processing of the message.
```

Comment:
after the processing of the message => after the message being processed.

Reply:
Both forms are clear in their meaning. 

One would most naturally use the  "being" form with something that is contemporary, like  "while being processed" the "has been" form would be most natural for after something that has finished e.g. has been processed. The term "the processing" hints a little more that it is the user's processing.

Reply:
I've tweaked it a little bit.